src/pages/configeditor/download_openshift.md

```diff
@@ -244,7 +244,7 @@ items:
 3. Create the proxy's deployment, proxy's service, and proxy's route:<br/>
   `oc create -f merative-acd-ce-oauth.yaml`<br/>
 
-   More options and details for the proxy are available at [OpenShift OAuth Proxy](https://github.com/openshift/oauth-proxy#openshift-oauth-proxy).  Information on troubleshooting the OAuth Proxy is found at [Troubleshooting the OAuth Proxy](/troubleshooting/troubleshooting-the-oauth-proxy/).  More details and options such as secret generation and cookie expiration are defined in the base [OAuth proxy](https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/overview/) docs.
+   More options and details for the proxy are available at [OAuth2 Proxy](https://oauth2-proxy.github.io/oauth2-proxy/docs/).  Information on troubleshooting the OAuth Proxy is found at [Troubleshooting the OAuth Proxy](/troubleshooting/troubleshooting-the-oauth-proxy/).
```

Comment:
... are available in the [OAuth2 Proxy docs]

Also, on the troubleshooting page, https://merative.github.io/acd-containers/troubleshooting/troubleshooting-the-oauth-proxy/, we should provide the commands in a consistent way. I think we've been using the black box vs. gray background. That page has both formats, so please update all to use the block with backticks before/after command.